crates/swc_ecma_preset_env/data/core-js-compat/data.json

```diff
@@ -1082,6 +1082,42 @@
     "safari": "10.0",
     "samsung": "5.0"
   },
+  "es.array.to-reversed": {
```

Comment:
Is this change intentional?
Seems like you invoked `yarn`

Reply:
done